code/R/model_pipeline.R

```diff
@@ -35,17 +35,19 @@
 ######################################################################
 #------------------------- DEFINE FUNCTION -------------------#
 ######################################################################
+source("code/R/tuning_grid.R")
+source("code/R/permutation_importance.R")
 
-
-pipeline <- function(data, model, split_number, perm=T, hyperparameters=NULL, outcome=NA){
+pipeline <- function(data, model, split_number, outcome=NA, hyperparameters=NULL, permutation=TRUE){
 
   # -----------------------Get outcome variable----------------------------->
   # If no outcome specified, use first column in data
+  print(paste0('data colnames ',colnames(data)))
```

Comment:
It looks like you wanted to see what the columns are, however, there are 6000 of them being printed. Do you think it would be better if we just print out the first column since it is being chosen as the outcome?

Reply:
Whoops, that was leftover from debugging. Removed!